### DISCUSSION TITLE
Question about prepping RNA-Seq samples

### DISCUSSION THREAD

**Discussion by josesanchez1815** [nih-cfde/training-and-engagement#447] [RNA - Seq] [answered]
These questions came up as we were discussing slides 25-26 of 91 from these slides https://osf.io/kj5av/
Will the robot prep from start to finish?

Does the PCR amplify the strands with adapters on it?

**Comment by ACharbonneau** [comment, marked as answer]
Different robots have different abilities so that's hard to answer.

For the PCR, yes. You make sticky ends on your fragmented sequences, ligate on adaptors, and those adaptors also serve as the PCR primer annealing locations
